src/components/InfoPanel/_info-panel.scss

```diff
@@ -51,9 +42,13 @@
       grid-template-columns: repeat(2, 1fr);
     }
   }
+}
 
-  .p-muted-heading {
-    font-size: 0.75rem;
-    margin-bottom: 0.25rem;
-  }
+.p-muted-heading {
+  font-size: 0.75rem !important;
+  margin-bottom: 0.25rem !important;
+}
+
+.p-modal .p-modal__dialog {
+  width: 90%;
```

Comment:
These should probably be scoped inside a class, something like `.info-panel__modal { ... }` and then we'd need to add the `info-panel__modal` class to the `<Modal>` component.

Reply:
Done! ✅  Also changed the location of `.p-muted-heading` to be inside `_entity-info.css` and slightly changed `_entity-info.css` to better represent the component hierarchy. Let me know if I should change it back.

Reply:
That looks good!